app/server/routes/v1/user.js

```diff
@@ -228,4 +228,53 @@ router.get("/:name", async (req, res) => {
   }
 });
 
+/**
+ * @swagger
+ * paths:
+ *  /api/user/{name}:
```

Comment:
do we need to identify a user with an guid/id here instead of name?

Reply:
We could but name is unique because name is the idir username. Maybe we rename 'name' to 'idir'?

Reply:
I'm making a ticket for the backlog to change 'name' to 'idir'. It will require changes across the whole backend as well as changes to the frontend api calls.